code/script.js

```diff
@@ -3,17 +3,92 @@
 // Step 1 - Welcome and introduction
 // Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`)
+  let messageName=prompt(`What is your name?`);
+  alert(`Welcome ${messageName}! Let's look at the menu!`);
 
 // Step 2 - Food choice
 // Your code goes here
+let foodChoice=prompt(`What would you like today: 
+1-Pizza
+2-Kebab
+3-Salad
+Please enter the number of your choice`);
+
+let selectedFood=""
+
+if(foodChoice==="1")
+{selectedFood="Pizza";}
+
+else if(foodChoice==="2")
+{selectedFood="Kebab";}
+
+else if(foodChoice==="3")
+{selectedFood="Salad";}
+
+else 
+{alert ("Oops, that is an invalid choiche, please select number 1, 2 or 3");}
```

Comment:
Maybe it would be interesting to add process.exit() for a clearer error handling?